src/sbt-test-0.13/sbt-ensime/multi-project-build/build.sbt

```diff
@@ -9,3 +9,5 @@ lazy val b = project.dependsOn(a)
 lazy val c = project.dependsOn(a).dependsOn(b % "test->test")
 
 lazy val d = project.dependsOn(a).dependsOn(b % "compile->compile;test->test")
+
+ensimeScalacOptions in a += "-wibble"
```

Comment:
also fails if `ensimeScalacOptions in a in Compile +=` is used.

Reply:
Is `ensimeScalaOptions in a += ...` a valid way to do that, I was able to get `ensimeScalaOptions in a in Compile += ...` to work

Reply:
interesting... in fact my use case at work has `in a in Compile` and it doesn't work so I'm surprised it worked for you. Are you sure you didn't run it the second time and the tests pass because of the mutated expectation file? (which is useful, but sometimes a real pain)

Reply:
It didn't, I made changes for it to work.

Reply:
But I was not sure the first approach is valid. I didn't want to go on a wild goose chase if it is not.

Reply:
Probably what is happening is sbt-ensime is defining the defaults at the (project, config) level which means any user settings at the (project) level are ignored. That's fairly standard in sbt. I'm happy to provide the config.

Reply:
Currently we are not looking at `ensimeScalacOptions` at the project level. 
Anyway, setting `scalacOptions in a += "-wibble"` is putting "-wibble" in all configurations of project a, so I expect `ensimeScalacOptions` should do the same.

Reply:
```
    ensimeScalacOptions := (
      (scalacOptions in Compile).value ++
      ensimeSuggestedScalacOptions((ensimeScalaVersion in ThisBuild).value)
    ).distinct,
```

I would have expected that to mean we're defining at the project level. Also, making our project level depend on the project's compile level is kind of weird. We should do one or the other... I'd be fine with per project/config level

Reply:
That definition was only used in the version 1.0
https://github.com/ensime/ensime-sbt/blob/2.0/src/main/scala/EnsimePlugin.scala#L343

In the version 2.0 format we are not considering `ensimeScalacOptions` at all
https://github.com/ensime/ensime-sbt/blob/2.0/src/main/scala/EnsimePlugin.scala#L343

Reply:
Nice! You found it :-)